.github/workflows/main.yml

```diff
@@ -3,6 +3,8 @@ name: Build conan-vs-extension
 on:
   push:
     branches: [ develop2 ]
+    tags:
+      - '*'
```

Comment:
probably we would like this to run only for some specific pattern of the tag. Maybe `v2*` or `2.*`?